[2023-01-24 02:42:46,381] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T01:19:00+00:00 [queued]>
[2023-01-24 02:42:46,404] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T01:19:00+00:00 [queued]>
[2023-01-24 02:42:46,404] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-24 02:42:46,404] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2023-01-24 02:42:46,405] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-24 02:42:46,438] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Leer_Datos_Respuesta_Nasa> on 2023-01-01 01:19:00+00:00
[2023-01-24 02:42:46,447] {standard_task_runner.py:52} INFO - Started process 1488 to run task
[2023-01-24 02:42:46,464] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Leer_Datos_Respuesta_Nasa', 'scheduled__2023-01-01T01:19:00+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpo1f86u_4', '--error-file', '/tmp/tmppy96txyi']
[2023-01-24 02:42:46,493] {standard_task_runner.py:80} INFO - Job 234: Subtask Leer_Datos_Respuesta_Nasa
[2023-01-24 02:42:46,673] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T01:19:00+00:00 [running]> on host fd9891b969b0
[2023-01-24 02:42:46,881] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Leer_Datos_Respuesta_Nasa
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T01:19:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T01:19:00+00:00
[2023-01-24 02:42:46,886] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-24 02:42:46,887] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /tmp && head /tmp/response_20230101.txt']
[2023-01-24 02:42:46,914] {subprocess.py:85} INFO - Output:
[2023-01-24 02:42:46,924] {subprocess.py:92} INFO - ***tmp0uotpneb
[2023-01-24 02:42:46,924] {subprocess.py:92} INFO - ***tmp1qsci6ey
[2023-01-24 02:42:46,924] {subprocess.py:92} INFO - ***tmp3mkzi6nc
[2023-01-24 02:42:46,924] {subprocess.py:92} INFO - ***tmp8dn6zzmk
[2023-01-24 02:42:46,924] {subprocess.py:92} INFO - ***tmpqkoon2cn
[2023-01-24 02:42:46,925] {subprocess.py:92} INFO - ***tmpt5bwpt7i
[2023-01-24 02:42:46,925] {subprocess.py:92} INFO - ***tmpwwsiiy_o
[2023-01-24 02:42:46,925] {subprocess.py:92} INFO - ***tmpx1fh85y4
[2023-01-24 02:42:46,925] {subprocess.py:92} INFO - pymp-igsi0xnb
[2023-01-24 02:42:46,925] {subprocess.py:92} INFO - response_20230101.txt
[2023-01-24 02:42:46,925] {subprocess.py:92} INFO - spacex_20230101.json
[2023-01-24 02:42:46,926] {subprocess.py:92} INFO - tmp2gofyf17
[2023-01-24 02:42:46,926] {subprocess.py:92} INFO - tmp_1gzauuq
[2023-01-24 02:42:46,926] {subprocess.py:92} INFO - tmpeo70ie_6
[2023-01-24 02:42:46,926] {subprocess.py:92} INFO - tmpeyj0vmxa
[2023-01-24 02:42:46,926] {subprocess.py:92} INFO - tmpjvr141a1
[2023-01-24 02:42:46,926] {subprocess.py:92} INFO - tmpnvc2r2ky
[2023-01-24 02:42:46,926] {subprocess.py:92} INFO - tmpo5sf5i7e
[2023-01-24 02:42:46,927] {subprocess.py:92} INFO - tmpoeosf_lq
[2023-01-24 02:42:46,927] {subprocess.py:92} INFO - tmppy96txyi
[2023-01-24 02:42:46,927] {subprocess.py:92} INFO - tmpqovrsavj
[2023-01-24 02:42:46,927] {subprocess.py:92} INFO - tmpxqudo0u_
[2023-01-24 02:42:46,930] {subprocess.py:92} INFO - Confirmaci√≥n de la NASA, pueden proceder
[2023-01-24 02:42:46,931] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-24 02:42:47,017] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Leer_Datos_Respuesta_Nasa, execution_date=20230101T011900, start_date=20230124T024246, end_date=20230124T024247
[2023-01-24 02:42:47,089] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-24 02:42:47,212] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
